### PR TITLE
Remove relative path info from executable field

### PR DESCRIPTION
**Pull request by wbrowne**
Related to https://github.com/grafana/grafana-zabbix/issues/1785 

[A change in the plugin SDK](https://github.com/grafana/grafana-plugin-sdk-go/pull/1007) allows us now to not require the nested datasource executable to live in the parent root directory.

### CODE REVIEW

File: src/datasource/plugin.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "annotations": true,
   "backend": true,
   "alerting": true,
-  "executable": "../gpx_zabbix-plugin",
+  "executable": "gpx_zabbix-plugin",
```
Comment (wbrowne) [inline src/datasource/plugin.json:9]: It might be a nice idea to also update this to 
```suggestion
  "executable": "gpx_zabbix-datasource",
```

**Comment by zoltanbedi** [comment]
@wbrowne I'm trying this out but it is not working for me. It tries to get the executable from the datasource folder. am I missing something?

I deleted the dist folder. run `mage -v` and then `yarn build`

**Comment by wbrowne** [comment]
@zoltanbedi could you take it for a spin now and let me know if it's okay for you? 🙏 Can you also confirm the directory structure of the built plugin looks like the following:

<details>
  <summary>Plugin dist</summary>
  
  ```
./go_plugin_build_manifest
./datasource
./datasource/query_help.md
./datasource/gpx_zabbix-datasource_windows_amd64.exe
./datasource/gpx_zabbix-datasource_darwin_amd64
./datasource/gpx_zabbix-datasource_linux_arm64
./datasource/module.js.map
./datasource/plugin.json
./datasource/img
./datasource/img/icn-zabbix-datasource.svg
./datasource/module.js
./datasource/dashboards
./datasource/dashboards/template_linux_server.json
./datasource/dashboards/zabbix_system_status.json
./datasource/dashboards/zabbix_server_dashboard.json
./datasource/gpx_zabbix-datasource_linux_arm
./datasource/gpx_zabbix-datasource_linux_amd64
./datasource/gpx_zabbix-datasource_darwin_arm64
./LICENSE
./panel-triggers
./panel-triggers/module.js.LICENSE.txt
./panel-triggers/module.js.map
./panel-triggers/plugin.json
./panel-triggers/img
./panel-triggers/img/icn-zabbix-problems-panel.svg
./panel-triggers/module.js
./CHANGELOG.md
./module.js.map
./plugin.json
./README.md
./img
./img/screenshot-annotations.png
./img/screenshot-dashboard01.png
./img/icn-zabbix-app.svg
./img/screenshot-triggers.png
./img/screenshot-showcase.png
./img/screenshot-metric_editor.png
./styles
./styles/dark.css
./styles/light.css
./module.js
  ```
</details>

**Review by zoltanbedi — APPROVED** [review]
LGTM, thanks @wbrowne!

**Pull request merged**